suite-common/sentry/src/index.ts

```diff
@@ -89,9 +89,8 @@ const ignoreErrors = [
     'device disconnected during action', // the same as with 'other call in progress'
 ];
 
-export const SENTRY_CONFIG: Options = {
+export const SENTRY_CONFIG = {
     dsn: 'https://6d91ca6e6a5d4de7b47989455858b5f6@o117836.ingest.sentry.io/5193825',
-    autoSessionTracking: false, // do not send analytical data to Sentry
     integrations: [
```

Comment:
To preserve the original behavior with session tracking disabled, you need to remove the default integration `browserSessionIntegration`, in my opinion. https://docs.sentry.io/platforms/javascript/guides/react/configuration/integrations/#removing-a-default-integration

https://docs.sentry.io/platforms/javascript/migration/v8-to-v9/#removed-apis

Reply:
Good point I was thinking it disabled by default.